app/src/main/java/com/ggetters/app/ui/central/views/PlayerProfileFragment.kt

```
@@ -148,6 +148,24 @@ class PlayerProfileFragment : Fragment() {
+                (activity as? HomeActivity)?.let { host ->
+                    // If this fragment was opened from a bottom-nav tab, pop the back stack first
+                    val fm = host.supportFragmentManager
+                    if (fm.backStackEntryCount > 0) fm.popBackStack()
+
+                    // Determine the desired tab based on where the user came from, if provided
+                    val originTabId = arguments?.getInt("origin_tab_id")
+                    if (originTabId != null) {
+                        host.navigateToTab(originTabId)
+                    } else {
+                        // Fallback: keep current tab selection, just finish this fragment
+                        requireActivity().onBackPressedDispatcher.onBackPressed()
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)